### ISSUE TITLE
Rework existing debugging lesson to use binding.irb

### ISSUE THREAD

**Issue by ChargrilledChook** [TheOdinProject/curriculum#25914]

**Comment by ChargrilledChook** [comment]
@JoshDevHub Just doing some admin to improve overall visiblity on this project

**Comment by ChargrilledChook** [comment]
@JoshDevHub Just doing some admin to improve overall visiblity on this project, would you mind replying here so I can assign this to you?

**Comment by JoshDevHub** [comment]
@ChargrilledChook Howdy!

**Comment by ChargrilledChook** [comment]
@joshdevhub

If we're going to tweak the positioning of this lesson, I'm happy to merge (but not publish) this lesson, maybe with a tweaked title.

Alternatively an epic branch might be easier easy to manage it

**Comment by JoshDevHub** [comment]
@ChargrilledChook

Yeah I'm in favor of moving it up. Before merging, I did want to look at maybe adding a bit more about `puts` vs. `p`. I should be able to get that done this weekend. After that, I'm fine with you merging it.
